### PR TITLE
feat(observer): implement `MutationObserver` in Angular

### PR DESCRIPTION
**Pull request by waterplea**

### CODE REVIEW

**Comment by waterplea** [comment]
Сук динамик

File: README.md [hunk 1]
```diff
@@ -28,7 +28,37 @@ npm i @ng-web-apis/mutation-observer
 
 ## Usage
 
-TBD
+Add `MutationObserverModule` and then use
+`(waMutationObserver)="onMutation($event)"` directive
+to watch mutation in an element.
+
+Alternatively use `MutationObserverService` manually to
```
Comment (MarsiBarsi) [inline README.md]: nit: you can specify here that the service is observable-based

File: projects/demo/src/app/app.component.ts [hunk 2]
```diff
@@ -6,4 +6,26 @@ import {ChangeDetectionStrategy, Component} from '@angular/core';
     styleUrls: ['./app.component.less'],
     changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class AppComponent {}
+export class AppComponent {
+    toggled = false;
+
+    disabled: true | null = null;
```
Comment (MarsiBarsi) [inline projects/demo/src/app/app.component.ts]: let's comment that it has true | null type cause it is an attribute in template

File: projects/mutation-observer/src/directives/test/mutation-observer.directive.spec.ts [hunk 3]
```diff
@@ -0,0 +1,86 @@
+import {Component} from '@angular/core';
```
Comment (MarsiBarsi) [inline projects/mutation-observer/src/directives/test/mutation-observer.directive.spec.ts:1]: sotka?

**Review by MarsiBarsi — APPROVED** [review]

**Pull request merged**